src/amr/data/particles/refine/split_1d.h

```diff
@@ -0,0 +1,56 @@
+#ifndef PHARE_SPLIT_1D_H
+#define PHARE_SPLIT_1D_H
+namespace PHARE::amr
+{
+template<size_t _interpOrder, size_t _refinedParticlesNbr>
+class ASplitter_1d : public ASplitter</*dim=*/1, _refinedParticlesNbr>
+{
+protected:
+    static constexpr size_t interpOrder = _interpOrder;
+};
+
+template<size_t _interpOrder, size_t _refinedParticlesNbr>
+class Splitter_1d : public ASplitter_1d<_interpOrder, _refinedParticlesNbr>
+{
+    Splitter_1d() /* This class should never be instantiated */ = delete;
+};
+
+template<>
+class Splitter_1d<1, 2> : public ASplitter_1d<1, 2>
+{
+public:
+    using Super                                 = ASplitter_1d<1, 2>;
+    static constexpr size_t dimension           = Super::dimension;
+    static constexpr size_t interpOrder         = Super::interpOrder;
+    static constexpr size_t refinedParticlesNbr = Super::refinedParticlesNbr;
+    using Super::deltas_;
+    using Super::weights_;
+
+    Splitter_1d()
+    {
+        for (auto& weight : weights_)
+            weight = weight_val_;
```

Comment:
all these lines could be shared with the other interp order... the only difference is going to be on the values of `deltas_[0][0]` and `deltas_[1][0]`. saying it differently, for a given number of babies, the pattern used to split is the same for all interporder... only weights and deltas are going to change from one case to the other.